src/EnergyPlus/InputProcessor.hh

```diff
@@ -802,17 +814,40 @@ namespace EnergyPlus {
 
 		static
 		void
+		IsNameEmpty(
+				std::string  & NameToVerify,
+				std::string const & StringToDisplay,
+				std::string const & NewName,
+				bool & ErrorFound );
+		static
+		void
+		IsNameEmpty(
+				std::string & NameToVerify,
+				std::string const & StringToDisplay,
+				bool & IsBlank,
+				bool & ErrorFound
+		);
+
+		static
+		void
+		IsNameEmpty(
+				std::string & NameToVerify,
+				std::string const & StringToDisplay,
+				bool & ErrorFound
+		);
+		static
+		void
 		RangeCheck(
-			bool & ErrorsFound, // Set to true if error detected
-			std::string const & WhatFieldString, // Descriptive field for string
-			std::string const & WhatObjectString, // Descriptive field for object, Zone Name, etc.
-			std::string const & ErrorLevel, // 'Warning','Severe','Fatal')
-			Optional_string_const LowerBoundString = _, // String for error message, if applicable
-			Optional_bool_const LowerBoundCondition = _, // Condition for error condition, if applicable
-			Optional_string_const UpperBoundString = _, // String for error message, if applicable
-			Optional_bool_const UpperBoundCondition = _, // Condition for error condition, if applicable
-			Optional_string_const ValueString = _, // Value with digits if to be displayed with error
-			Optional_string_const WhatObjectName = _ // ObjectName -- used for error messages
+				bool & ErrorsFound, // Set to true if error detected
```

Comment:
This is some kind of bug. There is nothing wrong with indention. I checked everything around those lines.